.github/workflows/central_publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Publish to the Ballerina central
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: choice
+        description: Select environment
+        required: true
+        options:
+          - CENTRAL
+          - DEV CENTRAL
+          - STAGE CENTRAL
+
+jobs:
+  central_publish:
+    name: Run central publish workflow
+    runs-on: ubuntu-latest
+
+    if: github.repository_owner == 'ballerina-platform'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17.0.7'
+
+      - name: Set Up Ballerina
+        uses: ballerina-platform/setup-ballerina@v1.1.0
+        with:
+          version: latest
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```

Comment:
We don't need this step. This is present in some workflow for historical reasons, but now the GitHub actions do have access to execute the Gradle scripts.

Reply:
As per the comment on line no. 43, we can remove this step altogether since we are using `gradle build` command when publishing, which should invoke the tests as well.